notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb

```diff
@@ -0,0 +1,1051 @@
+{
```

Comment:
<p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Changed 'Vertex SDK' to 'Vertex AI SDK'